msupply-foundation/open-msupply
# 2950 inbound return reason

Fixes #2950 


# 👩🏻‍💻 What does this PR do? 
 

Adds reason step to the inbound return from outbound shipment modal
![Screenshot 2024-02-15 at 2 49 01 PM](https://github.com/msupply-foundation/open-msupply/assets/55115239/e8922bb9-90d0-47e4-a3cd-dccdc2a019e4)


# 🧪 How has/should this change been tested? 

- Go to outbound shipment details page
- Select some outbound shipment lines
- Select process return action
- Click Ok & next
- See reason step
- Edit the reason & comment
- Ok button is enabled once you select a reason
- Mutation is called and modal closes on OK
